### DISCUSSION TITLE
我想知道chatglm3本地怎么进行多轮对话，并将对话喂到历史记录中？刚接触大模型不久，不太懂，谢谢大佬指点一下

### DISCUSSION THREAD

**Discussion by XuGuu** [zai-org/ChatGLM3#1024] [Q&A] [answered]
![17888f4ee9a8fa73812f84092fe411a](https://github.com/THUDM/ChatGLM3/assets/113074721/6cbdb1eb-22db-4d03-a4b2-8f0eab4075c4)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
可以查看一下 仓库中 prompt.md 这个文件是如何构造的
